std_msgs/Header header
int16[] data
int16 width
int16 height
int16 lam
int16 integration_time
float32[] central_wavelengths
float32[] qe
float32[] fwhm_nm